pyproject.toml

```diff
@@ -22,6 +22,7 @@ classifiers = [
   "Framework :: Django :: 3.2",
   "Framework :: Django :: 4.1",
   "Framework :: Django :: 4.2",
+  "Framework :: Django :: 5.0",
```

Comment:
By adding Python 3.12 to the test matrix you should also add the Python 3.12 classifier (adding it to the title of this PR should also be welcome).

Reply:
Ah yes! Thank you!